geth/api/backend.go

```diff
@@ -49,7 +48,7 @@ func NewStatusBackend() *StatusBackend {
 
 	nodeManager := node.NewManager()
 	accountManager := account.NewManager(nodeManager)
-	txQueueManager := transactions.NewManager(nodeManager)
+	txQueueManager := transactions.New(nodeManager)
```

Comment:
`transactions.New()` actually suggests that it's a new transaction is being created. I would keep `NewManager()` for this specific case.